adapters/persistence/adapter-maria/src/main/java/com/caregiver/domain/Group.java

```diff
@@ -0,0 +1,48 @@
+package com.caregiver.domain;
+
+import lombok.AccessLevel;
+import lombok.Builder;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+import java.util.ArrayList;
+import java.util.List;
+
+@Entity
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+@Builder
+public class Group {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "group_id")
+    private long id;
+
+    @Column(name = "group_name", nullable = false)
+    private String groupName;
+
+    @Column(name = "minimum_capacity", nullable = false)
+    private Integer minimumCapacity;
+
+    @Column(name = "maximum_capacity", nullable = false)
+    private Integer maximumCapacity;
+
+    @Column(name = "number_of_participants", nullable = false)
+    private Integer numberOfParticipants;
+
+    @Column(name = "description")
+    private String description;
+
+    @Column(name = "deleted", nullable = false)
+    private Boolean deleted;
```

Comment:
```suggestion
    private boolean isDeleted;
```
`boolean` 변수라면 앞에 is라는 접두어를 붙히는건 어떤가요?

Reply:
좋습니다. 컬럼은 그대로 두고 자바코드만을 말씀하시는 거죠~?

Reply:
음 저는 컬럼도 가능하면 맞추었으면 합니다.
아래와 같이요

`is_delete`

용진님께서 생각은 어떠신가요?

Reply:
강한 동의 💯